docker/.env

```diff
@@ -1,4 +1,3 @@
-COMPOSE_PROJECT_NAME=viame_web
```

Comment:
It's actually important that this line stay here, as otherwise the volume and service names will be wrong

Reply:
Right, I'll come up with something else.  Just a sec.